Finjector.Core/Domain/Coa.cs

```diff
@@ -22,9 +23,16 @@ public class Coa
 
         [Required]
         [MaxLength(128)] //Actually 70 I think, but a little play...
-        public string SegmentString { get; set; }
+        public string SegmentString { get; set; } = string.Empty;
 
         [MaxLength(3)]
         public string ChartType { get; set; } = string.Empty;
+
+        internal static void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            //Indexes?
+            modelBuilder.Entity<Coa>().HasIndex(a => a.FolderId);

```

Comment:
@jSylvestre i think this is redundant -- better to remove.  Because Coa has an FK to FolderId based on convention, EF will automatically create the FK, the constraint, and the FK index